core/templates/dev/head/css/oppia.css

```diff
@@ -3085,6 +3085,9 @@ md-card.preview-conversation-skin-supplemental-card {
   list-style-type: none;
   text-align: center;
 }
+.oppia-navbar-nav.oppia-navbar-profile > li:hover {
+  background-color: #009688
```

Comment:
Can you mention where you got this hex value from?

Reply:
@nithusha21 , @ankita240796  -- initially used color picker in inspector but it's also in css rule within oppia.css file, line 992.

`.navbar-light .navbar-container {
  background: #009688;
  height: 56px;
  width: 100%;
}`